pyodide-build/pyodide_build/pypabuild.py

```diff
@@ -62,7 +60,7 @@ def install_reqs(env: IsolatedEnv, reqs: set[str]):
     # only recythonize if it is present. We need them to always recythonize so
     # we always install cython. If the reqs included some cython version already
     # then this won't do anything.
-    env.install(["cython"])
+    env.install(["cython", "pythran"])
```

Comment:
If pythran is installed here, do we still need to install it in scipy?

Reply:
This change should be reverted for now because it doesn't actually work.

Reply:
Currently the issue is that we are using the `host_site_packages` one for the C include path but importing from the pypa/build virtual environment. The problem is that we have no obvious way of pointing the C include path into the pypa/build virtual environment.